src/main/java/frc/robot/subsystems/drive/PhoenixDrive.java

```diff
@@ -147,22 +149,20 @@ private void configurePathPlanner() {
                 this::getCurrentSpeeds,
                 (speeds) -> this.setControl(AutoRequest.withSpeeds(speeds)),
                 new HolonomicPathFollowerConfig(
-                        new PIDConstants(2),
-                        new PIDConstants(
-                                PhoenixDriveConstants.autoAlignmentkP,
-                                PhoenixDriveConstants.autoAlignmentkI,
-                                PhoenixDriveConstants.autoAlignmentkD),
+                        new PIDConstants(0),
+                        new PIDConstants(1, 0, 0),
```

Comment:
setting PIDConstants to 0 means we wouldn't follow the path, is this intended?

Reply:
thats a good question, i ran code with this constant set to 0 and it still ran correctly. This was something done by the previous 422 programmer that helped us out wednesday. I think he used global constraints in pathplanner (max velocity, accel) rather than the pid there. But if auto doesnt run for whatever reason, this is probably the first place to look for sure.

Reply:
Oh actually I remember what this does.

So if you set PID constants to 0 here it just means that if you fall behind the path or something that you are doomed to not catch up. The robot will still be fed the feedforward commands of the path but wouldn't be able to overcome any unexpected path deviations.

In general I don't think this is a sound approach at all, but we can leave it as is for now and tune this at competition or something